test/unit/server-serviceaccount.bats

```diff
@@ -2,6 +2,41 @@
 
 load _helpers
 
+@test "server/ServiceAccount: specify service account name" {
+  cd `chart_dir`
+
+  local actual=$( (helm template \
+      --show-only templates/server-serviceaccount.yaml  \
+      --set 'server.dev.enabled=true' \
+      --set 'server.serviceAccount.enabled=false' \
```

Comment:
I wonder if you could add a test or two that checks the serviceAccount in the statefulSet, when `server.serviceAccount.name` is set, and `server.serviceAccount.enabled` is `true` and `false`.

Reply:
Sure, will do 👍